apps/marketplace/components/BuyerRFX/BuyerRFXAdditionalInformationStage.js

```diff
@@ -93,6 +104,20 @@ class BuyerRFXAdditionalInformationStage extends Component {
             messages={{}}
           />
         </p>
+        <Textfield
+          model={`${this.props.model}.contactNumber`}
+          label="Contact number for Marketplace support"
+          description="This number will not be visible on the Digital Marketplace. It will only be used by the Marketplace operations team in case they need to contact you. Please include the area code."
+          name="contactNumber"
+          id="contactNumber"
+          htmlFor="contactNumber"
+          defaultValue={this.props[this.props.model].contactNumber}
+          maxLength={100}
+          validators={{
+            required,
+            validPhoneNumber
```

Comment:
When I did this, have a negative effect on the contact box and it would find a valid phone number 'wrong' i.e it would become red but it was fine to submitted it. (See image below) 
![image](https://user-images.githubusercontent.com/31645577/79167606-2c4a9400-7e2b-11ea-9843-6d6bd4a8d034.png)

Reply:
I can see the different behaviour that you're talking about.  Happy to leave as is but it looks like you might have committed my suggested changes in another file (check the specialist validators).